event_contact/tests/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
```

Comment:
Please don't remove others' copyright, just add yours below if you want. :wink:

BTW you can change this one to `Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>`